src/test/end-to-end/pages/newCase/newCase.js

```diff
@@ -3,10 +3,12 @@
 const testConfig = require('src/test/config');
 const newCaseConfig = require('./newCaseConfig');
 
-module.exports = async function () {
+module.exports = function () {
 
     const I = this;
 
-    await I.waitForText(newCaseConfig.waitForText, testConfig.TestTimeToWaitForText);
-    await I.waitForNavigationToComplete(newCaseConfig.locator);
+    I.waitForText(newCaseConfig.waitForText, testConfig.TestTimeToWaitForText);
+
```

Comment:
these 2 need to be async / await to wait for promise to resolve, possibly linked to removal of  waitForNavigation?